### PR TITLE
feat: ga

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/helpers/googleAnalytics.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  trackingElement.style.display = 'none';
+  document.body.appendChild(trackingElement);
+  document.body.removeChild(trackingElement);
+
+  return trackingElement;
+}
+
+export function sendHostname() {
+  const hostname = location.hostname;
+  const applicationKeyForStorage = `TOAST UI grid for ${hostname}: Statistics`;
```
Comment (jungeun-cho) [inline src/helpers/googleAnalytics.ts:41]: grid 대신에 chart로 바꿔주세요~

File: src/helpers/googleAnalytics.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+type GA_URL = 'https://www.google-analytics.com/collect';
+interface TrackingInfo {
+  v: number;
+  t: 'event';
+  tid: 'UA-129983528-2';
+  cid: string;
+  dp: string;
+  dh: 'grid';
+  el: 'grid';
```
Comment (jungeun-cho) [inline src/helpers/googleAnalytics.ts:9]: 여기도 'chart'가 맞아요~ appName들어가는부분!

File: src/helpers/googleAnalytics.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  setTimeout(() => {
+    if (document.readyState === 'interactive' || document.readyState === 'complete') {
+      imagePing('https://www.google-analytics.com/collect', {
+        v: 1,
+        t: 'event',
+        tid: 'UA-129983528-2',
+        cid: hostname,
+        dp: hostname,
+        dh: 'grid',
+        el: 'grid',
```
Comment (jungeun-cho) [inline src/helpers/googleAnalytics.ts:59]: 여기도 `'chart'`!

File: src/helpers/googleAnalytics.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
+type GA_URL = 'https://www.google-analytics.com/collect';
```
Comment (jungeun-cho) [inline src/helpers/googleAnalytics.ts:1]: GA_URL을 타입정의해서 사용하지 않아도 될 것 같아요.
imagePing 함수 내에서 한번만 사용해도되지않을까요?

**Review by jungeun-cho — COMMENTED** [review]
리뷰완료했습니다~!

**Review by shiren — APPROVED** [review]
복붙할때는 확인 철저.

**Review by lja1018 — APPROVED** [review]
리뷰완료
고생하셨습니다!

**Pull request merged**